src/pages/Signin/index.jsx

```diff
@@ -37,6 +40,23 @@ export class Signin extends React.Component {
     });
   };
 
+  responseGoogle = (response) => {
```

Comment:
You have implemented an identical function in Signup/index.js,
please consider refactoring to define this function once

Reply:
@ezrogha worked on feedback